torch/_dynamo/variables/tensor.py

```diff
@@ -1039,3 +1072,47 @@ def call_function(
             )
 
         return super().call_function(tx, args, kwargs)
+
+
+class TypedStorageVariable(VariableTracker):
+    def __init__(self, value, **kwargs):
+        self.value = value
+        super().__init__(**kwargs)
+
+    def reconstruct(self, codegen):
+        return super().reconstruct(codegen)
+
+    def as_proxy(self):
+        return self.proxy
+
+    def var_getattr(self, tx, name: str) -> VariableTracker:
+        if name == "_size":
+            return variables.LambdaVariable(
```

Comment:
This isn't right. If you getattr a method you get a bound method, this is not a lambda and they behave differently in different situations.

Reply:
This is the dynamo pattern for doing this - it doesn't create a lambda, its a lambda variable which, when invoked, calls method. (see the method below)